### PR TITLE
Suggest minor edit to CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by embray**
I love the note on here about impostor syndrome, as this is something I always try to communicate when disseminating the open source projects I work on to the user community.  Just a minor edit to this first sentence to make clear that even feeling like one can "write code" is not a prerequisite to contributing.  Of course this is already acknowledged in the following paragraph but I wanted to emphasize this earlier on.

[skip ci]

### CODE REVIEW

**Comment by pllim** [comment]
The text was originally developed by https://github.com/Unidata/MetPy and added via #8665 . I'll have to punt this to @astropy/coordinators .

**Comment by pllim** [comment]
This shouldn't haven been open so long. Does the comment in https://github.com/astropy/astropy/pull/10507#pullrequestreview-434128982 still need addressing?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -22,11 +22,12 @@ ready to be an open source contributor; that your skills aren't nearly good
 enough to contribute. What could you possibly offer a project like this one?
 
 We assure you - the little voice in your head is wrong. If you can write code at
```
Comment (kelle) [inline CONTRIBUTING.md:24]:
```suggestion
We assure you - the little voice in your head is wrong. If you can write code or documentation,
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -22,11 +22,12 @@ ready to be an open source contributor; that your skills aren't nearly good
...
-all, you can contribute code to open source. Contributing to open source
-projects is a fantastic way to advance one's coding skills. Writing perfect code
-isn't the measure of a good developer (that would disqualify all of us!); it's
-trying to create something, making mistakes, and learning from those
-mistakes. That's how we all improve, and we are happy to help others learn.
+all, or even documentation, you can contribute code to open source.
```
Comment (kelle) [inline CONTRIBUTING.md:25]:
```suggestion
you can contribute code to open source.
```

**Comment by kelle** [comment]
I agree with @mhvk , the title "Contributing Code" can just be removed. I have some other small changes I would make to this text. I will try to add them to @embray's branch.

**Comment by embray** [comment]
@kelle Did you commit the suggestions?   I don't see any indication that you did.  This is weird: my e-mail notifications show a bunch of change suggestions you made, but they seem to have disappeared from the PR here on the website.

Neverthless since I have them in my inbox I can make the changes.

**Comment by kelle** [comment]
I made a PR to the patch-3 branch of embray's fork and am feeling like a GitHub superuser. 💪🏼

**Comment by embray** [comment]
I incorporated @kelle's changes.  I think this is finally (nearly a year later?) ready?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -176,3 +145,32 @@ package.
...
+  not yet ready for the tests to run. For example:
+
+      $ git commit -m "WIP widget [ci skip]"
+
+  - If you already made the commit without including this string, you can edit
+    your existing commit message by running:
+
+        $ git commit --amend
+
+- To skip only the tests running on Travis CI use ``[skip travis]``.
```
Comment (mhvk) [inline CONTRIBUTING.md:163]: Delete these travis related instructions.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -176,3 +145,32 @@ package.
...
+
+- If your commit makes substantial changes to the documentation but no code
```
Comment (mhvk) [inline CONTRIBUTING.md:165]: Maybe combine this and the last point in a shortened one, and call it 'skip ci'?

**Review by mhvk — COMMENTED** [review]
@embray - mostly looks good, but the tips now moved to the end contain some outdated stuff...

File: CONTRIBUTING.md (hunk 3 above)
Comment (embray) [reply CONTRIBUTING.md:163]: Oof, good catch.

File: CONTRIBUTING.md (hunk 4 above)
Comment (embray) [reply CONTRIBUTING.md:165]: It really seems like this whole section has a lot of redundancy, and can be reduced to a single bullet-point about skip CI.

Comment (embray) [reply CONTRIBUTING.md]: see 36ec924

Comment (embray) [reply CONTRIBUTING.md]: Oops, scratch that.  Just realized that section has already been rewritten in main, so I'm just rebasing and keeping the existing text that's there.

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -185,3 +160,36 @@ package.
...
+  you are not yet ready for the tests to run. For example:
+
+      $ git commit -m "WIP widget [ci skip]"
+
+  - If you already made the commit without including this string, you can edit
+    your existing commit message by running:
+
+        $ git commit --amend
+
+- Unfortunately, GitHub Actions ignores ``[ci skip]`` for a PR, so we recommend
```
Comment (mhvk) [inline CONTRIBUTING.md:180]: I think only circle CI and readthedocs now ignore `skip ci`?

**Comment by mhvk** [comment]
@embray - right now, only an inconsistency is left. Though I agree with you that the `skip ci` text is now very repetitive. If you can shorten it, that would be great.

File: CONTRIBUTING.md (hunk 5 above)
Comment (embray) [reply CONTRIBUTING.md:180]: It appears so.  I'll fix that.

Comment (embray) [reply CONTRIBUTING.md]: Ok, I went ahead and removed that bullet point since it's no longer really pertinent.

**Review by mhvk — APPROVED** [review]
Not sure exactly what we were waiting for here - I just looked again and it looks good! So I'm approving. @embray - any reason not to merge?

**Comment by kelle** [comment]
LGTM!

**Comment by pllim** [comment]
@meeseeksdev backport to v5.0.x

**Pull request merged**